aspnetcore/data/ef-rp/complex-data-model.md

```diff
@@ -129,14 +129,17 @@ With the `[Column]` attribute, `Student.FirstMidName` in the data model maps to
 
 The `Required` attribute makes the name properties required fields. The `Required` attribute isn't needed for non-nullable types such as value types (for example, `DateTime`, `int`, and `double`). Types that can't be null are automatically treated as required fields.
 
-The `Required` attribute could be replaced with a minimum length parameter in the `StringLength` attribute:
+The `Required` attribute must be used with `MinimumLength` for the `MinimumLength` to be enforced.
```

Comment:
```suggestion
The `StringLength` attribute specifies maximum length and can optionally specify `MinimumLength`.
```

Required attribute doesn't have to be used with minlength -- if minlength is 2, no Required attr., and they enter 1 character, minlength would be enforced.